decred/tests/unit/crypto/secp256k1/test_curve.py

```diff
@@ -556,8 +581,7 @@ def test_scalar_mult_rand(randBytes):
     # and the new random exponent together (mod N).
     x, y = curve_obj.Gx, curve_obj.Gy
     exponent = 1
-    # 16 instead of 1024 because CPU-intensive.
-    for _ in range(16):
+    for _ in range(1024):
```

Comment:
I had lowered this down because this test was too slow, but the optimization in #160 allows us to put it back as it was.